VariantEncoding.md

```diff
@@ -423,14 +451,36 @@ Field names are case-sensitive.
 Field names are required to be unique for each object.
 It is an error for an object to contain two fields with the same name, whether or not they have distinct dictionary IDs.
 
-# Versions and extensions
+## Versions and extensions
 
 An implementation is not expected to parse a Variant value whose metadata version is higher than the version supported by the implementation.
 However, new types may be added to the specification without incrementing the version ID.
 In such a situation, an implementation should be able to read the rest of the Variant value if desired.
 
-# Shredding
+## Shredding
 
 A single Variant object may have poor read performance when only a small subset of fields are needed.
 A better approach is to create separate columns for individual fields, referred to as shredding or subcolumnarization.
 [VariantShredding.md](VariantShredding.md) describes the Variant shredding specification in Parquet.
+
+## Conversion to JSON
+
+Values stored in the Variant encoding are a superset of JSON values.
+For example, a Variant value can be a date that has no equivalent type in JSON.
+To maximize compatibility with readers that can process JSON but not Variant, the following conversions should be used when producing JSON from a Variant:
+
+| Variant type  | JSON type | Representation requirements                              | Example                              |
+|---------------|-----------|----------------------------------------------------------|--------------------------------------|
+| Null type     | null      | `null`                                                   | `null`                               |
+| Boolean       | boolean   | `true` or `false`                                        | `true`                               |
+| Exact Numeric | number    | Digits in fraction must match scale, no exponent         | `34`, 34.00                          |
+| Float         | number    | Fraction must be present                                 | `14.20`                              |
```

Comment:
it might be worth noting that relying on the json values here might be lossy for some numbers (this is implied but people still ask questions ...)